src/main/java/com/patchworkmc/mapping/remapper/ManifestRemapper.java

```diff
@@ -21,19 +22,22 @@ public String remapMemberDescription(String descriptor) {
 		return asmRemapper.mapDesc(descriptor);
 	}
 
+	/**
+	 * AccessTransformers lack descriptors for fields, so we have to use a NaiveRemapper instead.
```

Comment:
Huh? It looks like we just use ignoreFieldDesc here, NaiveRemapper isn't involved.

Reply:
Old docs describing impl. details I hadn't fully decided yet.